src/api/custom-resolvers/mutations/verify-account-email-address/index.ts

```diff
@@ -44,24 +43,7 @@ const verifyAccountEmailAddress = async (root: any, variables: VerifyEmailAddres
     }
 
     const { id } = account;
-    const { id: statusId } = account.status;
-
-    /**
-     * Check if the verification period has expired.
-     * If so, return success=false, expired=true
-     */
-    const now = new Date();
-    const canActivateAccount = isBefore(now, account[VERIFICATION_EXPIRY]);
-
-    if (!canActivateAccount) {
-      console.info('Unable to verify account email address - verification period has expired');
-
-      return {
-        expired: true,
-        success: false,
-        accountId: id,
-      };
-    }
```

Comment:
Similar to above, I don't think we should be removing such logic.

We shouldn't rely entirely on the CRON jobs. If for some reason a job doesn't execute successfully, accounts things could happen that shouldn't.